packages/test-base/src/commonTest/kotlin/io/realm/kotlin/test/common/RealmDictionaryTests.kt

```diff
@@ -3063,6 +3078,10 @@ internal class RealmAnyDictionaryTester(
                 assertEquals(expectedObj.stringField, assertNotNull(actualObj).stringField)
             }
             null -> assertNull(actualValue)
+            // FIXME Should we rather test nested collections somewhere else?
```

Comment:
I don't have a strong preference either way.

Reply:
Added a no-op and pointed to the separate tests in `RealmAnyNestedCollectionTests`